librz/core/cmd_descs/cmd_search.yaml

```diff
@@ -3,8 +3,173 @@
 ---
 name: cmd_search
 commands:
+  - name: "/a"
+    summary: Assemble the opcodes and search its bytes
+    cname: cmd_search_assemble
+    type: RZ_CMD_DESC_TYPE_ARGV_STATE
+    default_mode: RZ_OUTPUT_MODE_STANDARD
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_JSON
+      - RZ_OUTPUT_MODE_QUIET
+      - RZ_OUTPUT_MODE_TABLE
+    args:
+      - name: opcodes
+        type: RZ_CMD_ARG_TYPE_STRING
+
+  - name: "/c"
```

Comment:
I was thinking to change this and have one command which search for all or one specific material

Reply:
Sure. But with the next PR. This is just so everything builds and in the yaml files.

Reply:
I will do this. i think what will be done for this is just `/c <type>` where by default is going to be `all` or if specified, whatever the user will choose.